src/reachy2_sdk/orbita/orbita.py

```diff
@@ -107,40 +104,37 @@ def turn_on(self) -> None:
         """Turn all motors of the orbita2d on.
         All orbita2d's motors will then be stiff.
         """
-        self._compliant = False
+        self._state["compliant"] = False
+
+        async def set_in_loop() -> None:
+            self._register_needing_sync.append("compliant")
+            self._need_sync.set()
+
+        fut = asyncio.run_coroutine_threadsafe(set_in_loop(), self._loop)
+        fut.result()
 
     def turn_off(self) -> None:
         """Turn all motors of the orbita2d on.
         All orbita2d's motors will then be stiff.
         """
-        self._compliant = True
+        self._state["compliant"] = True
+
+        async def set_in_loop() -> None:
+            self._register_needing_sync.append("compliant")
+            self._need_sync.set()
+
+        fut = asyncio.run_coroutine_threadsafe(set_in_loop(), self._loop)
+        fut.result()
 
     def is_on(self) -> Any:
         """Get compliancy of the actuator"""
         return not self._compliant
 
     @property
-    def temperatures(self) -> Dict[str, Register]:
+    def temperatures(self) -> Dict[str, Any]:
```

Comment:
Avoid the use of Any. mypy will do a better job knowing the exact type. I believe we have a float here